compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -361,7 +361,7 @@ cmdInit numProcessors =
 cmdPackage :: Int -> Mod CommandFields Command
 cmdPackage numProcessors =
     command "package" $ info (helper <*> cmd) $
-       progDesc "Compile the DAML program into a DAML Archive (DAR)"
+       progDesc "Compile the DAML program into a DAR (deprecated)"
```

Comment:
```suggestion
       progDesc "Deprecated. Compile the DAML program into a DAR"
```
This is consistent with the format @bame-da introduced in https://github.com/digital-asset/daml/pull/7458/files.